devtools/riscv64.patch

```diff
@@ -1,54 +1,46 @@
-Index: PKGBUILD
-===================================================================
---- PKGBUILD	(revision 433967)
-+++ PKGBUILD	(working copy)
+diff --git PKGBUILD PKGBUILD
+index 6cc7428..0de487b 100644
+--- PKGBUILD
++++ PKGBUILD
 @@ -3,9 +3,9 @@
  
  pkgname=devtools
- pkgver=20211129
+ pkgver=20220126
 -pkgrel=1
-+pkgrel=1.2
++pkgrel=1.1
  pkgdesc='Tools for Arch Linux package maintainers'
 -arch=('any')
 +arch=('x86_64')
  license=('GPL')
- url='https://github.com/archlinux/devtools'
+ url='https://gitlab.archlinux.org/archlinux/devtools'
  depends=('bash' 'openssh' 'subversion' 'rsync' 'arch-install-scripts'
-@@ -12,7 +12,9 @@
-          'git' 'bzr' 'mercurial' 'diffutils' 'util-linux' 'awk')
+@@ -13,7 +13,8 @@
  makedepends=('asciidoc')
  optdepends=('btrfs-progs: btrfs support')
--source=(https://sources.archlinux.org/other/${pkgname}/${pkgname}-${pkgver}.tar.gz{,.sig})
-+source=(https://sources.archlinux.org/other/${pkgname}/${pkgname}-${pkgver}.tar.gz{,.sig}
-+        https://github.com/archlinux/devtools/pull/64.patch
```

Comment:
Thanks for the update. As we are no longer patching the devtools code now, we should be splitting the additional files to a new package like `devtools-riscv64` so that we are no longer blocked by new devtools updates.